azure-pipelines.yml

```diff
@@ -36,7 +36,7 @@ steps:
 - script: |
     export JAVA_HOME=/usr/lib/jvm/temurin-17-jdk-amd64/
     ./gradlew -version
-    ./gradlew test jacocoMergedReport checkFormat spotlessCheck 
+    ./gradlew test jacocoMergedReport
```

Comment:
removed ?

Reply:
only from the azure pipeline to make it faster, unncessery change for this PR anyway so will remove